### PR TITLE
Feat/10.4/create logic for update vet info

### PR DESCRIPTION
**Pull request by KobzarMax**

### CODE REVIEW

File: client/src/app/admin-page/components/admin-edit-vet/admin-edit-vet.component.html [hunk 1]
```diff
@@ -1,53 +1,39 @@
...
       </div>
 
-      <table mat-table [dataSource]="dataSource" class="mat-elevation-z8 demo-table" matSort>
-        <!-- Position Column -->
+
+      <table mat-table [dataSource]="dataSource" class="mat-elevation-z8 demo-table">
         <ng-container matColumnDef="id">
-          <th mat-header-cell *matHeaderCellDef mat-sort-header>ID</th>
-          <td mat-cell *matCellDef="let element">{{element.id}}</td>
+            <th mat-header-cell *matHeaderCellDef>No.</th>
```
Comment (sanderpa) [inline client/src/app/admin-page/components/admin-edit-vet/admin-edit-vet.component.html]: It must be ID in this place

File: client/src/app/admin-page/components/admin-edit-vet/admin-edit-vet.component.html [hunk 2]
```diff
@@ -1,53 +1,39 @@
...
-          </td>
+            <th mat-header-cell *matHeaderCellDef>Назва клініки</th>
+            <td mat-cell *matCellDef="let element">{{element.title}}</td>
         </ng-container>
 
-        <!-- Adress Column -->
         <ng-container matColumnDef="adress">
-          <th mat-header-cell *matHeaderCellDef mat-sort-header>Адреса</th>
-          <td mat-cell *matCellDef="let element">{{element.adress}}</td>
+            <th mat-header-cell *matHeaderCellDef>Адреса</th>
```
Comment (sanderpa) [inline client/src/app/admin-page/components/admin-edit-vet/admin-edit-vet.component.html]: missing mat-sort-header atribute

File: client/src/app/admin-page/components/admin-edit-vet/admin-edit-vet.component.html [hunk 3]
```diff
@@ -57,19 +43,15 @@
...
-        <tr mat-row *matRowDef="let row; columns: displayedColumns;"></tr>
-
       </table>
 
-      <mat-paginator [pageSizeOptions]="[10, 25, 50, 100]" aria-label="Select page of users"></mat-paginator>
-
-    </mat-tab>
-
-  </mat-tab-group>
+      <mat-paginator [pageSizeOptions]="[5, 10, 20, 50]" aria-label="Select page of users"></mat-paginator>
```
Comment (sanderpa) [inline client/src/app/admin-page/components/admin-edit-vet/admin-edit-vet.component.html]: replace start pagination of 10 elements

File: client/src/app/admin-page/components/admin-edit-pet/admin-edit-pet.component.ts [hunk 4]
```diff
@@ -101,6 +101,7 @@ export class AdminEditPetComponent implements OnInit, AfterViewInit {
 			this.dataSource.paginator.firstPage()
 		}
 	}
+	onEdit() {}
```
Comment (sanderpa) [inline client/src/app/admin-page/components/admin-edit-pet/admin-edit-pet.component.ts]: what means this empty method?

File: client/src/app/admin-page/components/admin-edit-vet/admin-edit-vet.component.ts [hunk 5]
```diff
@@ -53,6 +53,21 @@ export class AdminEditVetComponent implements OnInit, AfterViewInit {
 		}
 	}
 
+	getVets() {
```
Comment (sanderpa) [inline client/src/app/admin-page/components/admin-edit-vet/admin-edit-vet.component.ts]: Access modifiers need to be added

File: client/src/app/admin-page/components/admin-edit-vet/admin-edit-vet.component.ts [hunk 6]
```diff
@@ -53,6 +53,21 @@ export class AdminEditVetComponent implements OnInit, AfterViewInit {
...
+		this.apiServices.getVetDetails().subscribe(response => {
+			this.dataSource.data = response
+		})
+	}
+
+	openDialog(row: IVetsUnitInfo) {
```
Comment (sanderpa) [inline client/src/app/admin-page/components/admin-edit-vet/admin-edit-vet.component.ts]: Access modifiers need to be added

File: client/src/app/admin-page/components/admin-edit-vet/admin-edit-vet.component.ts [hunk 7]
```diff
@@ -62,13 +77,7 @@ export class AdminEditVetComponent implements OnInit, AfterViewInit {
...
 	}
 
-	private getVets() {
-		this.apiServices.getVetDetails().subscribe(response => {
-			this.dataSource.data = response
-		})
-	}
-
-	public onOpenAddDialog(): void {
+	onOpenAddDialog(): void {
```
Comment (sanderpa) [inline client/src/app/admin-page/components/admin-edit-vet/admin-edit-vet.component.ts]: Access modifiers need to be added

File: client/src/app/admin-page/components/admin-edit-vet/admin-edit-vet.component.ts [hunk 8]
```diff
@@ -82,7 +91,7 @@ export class AdminEditVetComponent implements OnInit, AfterViewInit {
 		})
 	}
 
-	public onOpenDeleteDialog(id: number): void {
+	onOpenDeleteDialog(id: number): void {
```
Comment (sanderpa) [inline client/src/app/admin-page/components/admin-edit-vet/admin-edit-vet.component.ts]: Access modifiers need to be added

File: client/src/app/admin-page/components/admin-edit-vet/admin-edit-vet.component.ts [hunk 9]
```diff
@@ -95,7 +104,7 @@ export class AdminEditVetComponent implements OnInit, AfterViewInit {
 		})
 	}
 
-	private onDeleteClinic(id: number): void {
+	onDeleteClinic(id: number): void {
```
Comment (sanderpa) [inline client/src/app/admin-page/components/admin-edit-vet/admin-edit-vet.component.ts]: Access modifiers need to be added

**Review by sanderpa — CHANGES_REQUESTED** [review]

File: client/src/app/admin-page/components/admin-edit-vet/admin-edit-vet.component.html (hunk 2 above)
Comment (Samwa-ua) [reply client/src/app/admin-page/components/admin-edit-vet/admin-edit-vet.component.html]: Not all columns need to be sorted

File: client/src/app/admin-page/components/admin-edit-vet/admin-edit-vet.component.ts [hunk 10]
```diff
@@ -20,9 +19,10 @@ import { VetDeleteDialogComponent } from '../vet-delete-dialog/vet-delete-dialog
 export class AdminEditVetComponent implements OnInit, AfterViewInit {
 	selectFormControl = new FormControl('', Validators.required)
 
-	displayedColumns: string[] = ['id', 'title', 'adress', 'phone', 'email', 'delete']
+	displayedColumns: string[] = ['id', 'title', 'adress', 'details', 'delete']
+	IVetsInfo$: Observable<Array<IVetsUnitInfo>>
```
Comment (Samwa-ua) [inline client/src/app/admin-page/components/admin-edit-vet/admin-edit-vet.component.ts]: Where did you use this observable ?

File: client/src/app/admin-page/components/update-vet/update-vet.component.html [hunk 11]
```diff
@@ -0,0 +1,38 @@
+<div class="basic-container">
+  <form class="example-form" (ngSubmit)='onUpdateVet(f.value)' #f="ngForm">
+    <mat-form-field class="example-full-width id-hide" appearance="fill">
+      <mat-label>No</mat-label>
```
Comment (Samwa-ua) [inline client/src/app/admin-page/components/update-vet/update-vet.component.html:4]: ID?

**Review by Samwa-ua — APPROVED** [review]

**Pull request closed without merging**